src/services/Shell/Shell.js

```diff
@@ -63,7 +63,7 @@ function Shell() {
         } catch (e) {
             console.error(e);
             active = false;
-            error = new Error(e);
+            error = new Error(e, { cause: 'Failed to initialize shell transport' });
```

Comment:
Those should be swapped https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause#rethrowing_an_error_with_a_cause
```suggestion
            error = new Error('Failed to initialize shell transport', { cause: e });
```

Reply:
Thank you for the suggestion!